### DISCUSSION TITLE
Next GE meeting: January 9?

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#95] [Team Posts]
As this coming Monday is, or *should be*, a holiday, I suggest we meet next time on January 9th? The same time (<s>18:30</s> 17:30 UTC) still suit people?..

**Comment by fcbond** [comment]
Sounds good to me.

On Wed, 21 Dec 2022 at 17:12, Olga Zamaraeva ***@***.***>
wrote:

> As this coming Monday is, or *should be*, a holiday, I suggest we meet
> next time on January 9th? The same time (18:30 UTC) still suit people?..
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/40>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRX3ICVMIO6KQIV5GGTWOMT7RANCNFSM6AAAAAATFZJSUQ>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by emilymbender** [comment]
Should work for me!

**Comment by olzama** [comment]
@delph-in/participants Let's try tomorrow then? 17:30 UTC still good for you, @wsun106 , to start the Zoom meeting? If not, just let us know :).

**Comment by olzama** [comment]
I just noticed in the original post I said 18:30 UTC; that's not correct as all the previous meetings were 17:30 UTC. But let's see what people prefer in this New Year :).

**Comment by fcbond** [comment]
Either time is ok for me.

On Sun, 8 Jan 2023 at 13:35, Olga Zamaraeva ***@***.***>
wrote:

> I just noticed in the original post I said 18:30 UTC; that's not correct
> as all the previous meetings were 17:30 UTC. But let's see what people
> prefer in this New Year :).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/40/comments/4>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRRI66LQ4BTHPY7QAFTWRKYB5ANCNFSM6AAAAAATFZJSUQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by wsun106** [comment]
Apologies for my belated reply. i am away from my emails last weekend. I can do either time but prefer 18.30pm slightly.
Look forward to the first meeting in 2023!


https://cl-cam-ac-uk.zoom.us/j/92649124250?pwd=L2YzUE1pU3NHSmlsVDVnS203VmpNdz09


Conference ID: 926 4912 4250
password: 674288



At 2023-01-08 20:33:46, "Olga Zamaraeva" ***@***.***> wrote:

@delph-in/participants Let's try tomorrow then? 17:30 UTC still good for you, @wsun106 , to start the Zoom meeting? If not, just let us know :).

**Comment by olzama** [comment]
Let's try to do ***17:30 UTC*** today, as we have been doing before, but maybe starting next time we could switch to a later time, depending on everyone's preferences?

**Comment by wsun106** [comment]
yes! let's talk at 17.30 today. W




在 2023-01-09 18:35:57，"Olga Zamaraeva" ***@***.***> 写道：

Let's try to do 17:30 UTC today, as we have been doing before, but maybe starting next time we could switch to a later time, depending on everyone's preferences?

**Comment by olzama** [comment]
@wsun106 mentioned it would be better for her if we started an hour later (18:30 UTC). For me, it would still be OK but I am not sure we would keep our American participants this way?.. It would be 10:30 in Seatlle for example. @emilymbender @danflick what do you say? If it makes its harder for you, we'll keep 17:30 UTC since @wsun106 said it was still possible for her as well.

**Comment by emilymbender** [comment]
Are we on an every-other-Monday schedule? On Jan 23, I have a conflict from 11, Feb 6 I'm traveling, and Feb 20 is a holiday here...

**Comment by olzama** [comment]
Yeah, in principle, every other Monday. So, what matters is how likely you are to make it in general, occasional conflicts will of course happen. If moving it an hour later increases the likelihood of occasional conflicts significantly though, that's another matter :). (And that's what I suspected, as 10:30 is kind of already the middle of a working morning where you are :) ).

**Comment by emilymbender** [comment]
I don't have a regular conflict at that time this quarter, and if I put it on my calendar as recurring, I can avoid conflicts in Spring though. (Also makes it easier to go for a run first :)

**Comment by olzama** [comment]
Great, thanks, let's hear from @danflick @fcbond @arademaker and any other interested parties :)

**Comment by fcbond** [comment]
I'm in Europe, so either time works for me.

On Tue, 10 Jan 2023 at 16:20, Olga Zamaraeva ***@***.***>
wrote:

> Great, thanks, let's hear from @danflick <https://github.com/danflick>
> @fcbond <https://github.com/fcbond> @arademaker
> <https://github.com/arademaker> and any other interested parties :)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/delph-in/teams/participants/discussions/40/comments/14>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAIPZRUB5JOMMZGXFYGJP5LWRV44PANCNFSM6AAAAAATFZJSUQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Francis Bond <https://fcbond.github.io/>

**Comment by danflick** [comment]
Either time is fine for me, too.

 Dan
